README.md

```diff
@@ -1,2 +1,91 @@
-# tw-academy-s9
-This is my repository for the Lesson 9 - TW Academy
+# Sample Style Guide
+
+## Technical Writing Academy - May 2021
+
+## Table of Contents
+
+* __Bold__
+* __Italics__
+* __Fixed Font__
+* __Abbreviation__
+* __Capitalization__
+* __Acronyms__
+* __Commas in Lists__
+* __Numbers__
+* __Contractions__
+* __Apostrophes__
+
+## Bold
+
+Use Bold to highlight button names, fields names, menus, radio buttons, or checkboxes. For example:
+
+* Click the **Create** button to start the process.
+* Enter your IP address in the **IP Address** field.
+
+## Italics
+
+Use Italics to emphasize a word or group of words. For example:
+* *At least* two reviews are required before you check in your code.
+* You must *not* enter without your ID card.
+
+**IMPORTANT**: Use this style sparingly to avoid diluting its strength.
+
+## Fixed Font
+
+Use a fixed font to highlight:
+* **Commands or pieces of code**. For example:
+    Run the `nmp install text-cli` command to install the utility.
+* **File or folder names**. For example:
+    Read the `README.md` file before you proceed.
+
+## Abbreviations
+
+Never use the following abbreviations in a formal document:
+
+| Abbreviation | Instead use |
+| --- | --- |
+| e.g. | Insted use "for example". |
```

Comment:
Instead has a typo.

Also, because you put "Instead use" in the header, you could remove the "Instead" word on the cells of these columns.